syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.airline_app";
option java_outer_classname = "AirlineAppProto";
option objc_class_prefix = "HLW";

package airline_app;

//The request message containing the selection
message Selection {
    string value = 1;
}

//The response message from the system
message SysMessage {
    string message = 1;
}

//Services provided as List, Reserve and Display
service AirlineApplication {
    rpc AirlineList (Selection) returns (SysMessage) {}
    rpc AirlineReserv (Selection) returns (SysMessage) {} 
    rpc AirlineDisplay (Selection) returns (SysMessage) {}
}